src/pages/dashboard/programme/index.page.tsx

```diff
@@ -0,0 +1,217 @@
+  TOTAL_NUMBER_OF_SITES,
+  TOTAL_VOLUNTEERS,
+  VOLUNTEERS_CREATED_BY_AGE,
+  VOLUNTEERS_CREATED_BY_GENDER
+} from "../mockedData/dashboard";
+
+export interface DashboardTableDataProps {
+  label: string;
+  valueText: string;
+  value: number;
+}
+
+export interface GraphicLegendProps {
+  label: string;
+  value: string;
+  color: string;
+}
+
+export interface DashboardDataProps {
+  value?: string;
+  unit?: string;
+  secondValue?: string;
+  graphic?: string;
+  tableData?: DashboardTableDataProps[];
+  maxValue?: number;
+  graphicLegend?: GraphicLegendProps[];
+  graphicTargetLandUseTypes?: DashboardTableDataProps[];
+}
+
+const Dashboard = () => {
+  const t = useT();
+  const dataToggle = ["Absolute", "Relative"];
+  const dataToggleGraphic = ["Table", "Graphic"];
+  const sharedRef = useContext(RefContext);
+  const dashboardHeader = [
+    {
+      label: "Trees Planted",
+      value: "12.2M"
+    },
+    {
+      label: "Hectares Under Restoration",
+      value: "5,220 ha"
+    },
+    {
+      label: "Jobs Created",
+      value: "23,000"
+    }
+  ];
+
+  return (
+    <div className="flex flex-1 gap-4 overflow-hidden bg-neutral-70 p-4 ">
+      <div ref={sharedRef} className="w-3/5 overflow-auto pr-2 ">
+        <PageRow className="gap-4 p-0">
+          <div className="grid w-full grid-cols-3 gap-4">
+            {dashboardHeader.map((item, index) => (
+              <div key={index} className="rounded-lg bg-white px-5 py-4.5">
+                <Text variant="text-10-light" className="text-darkCustom opacity-60">
+                  {item.label}
+                </Text>
+
+                <div className="flex items-center gap-2">
+                  <Text variant="text-20" className="text-darkCustom" as="span">
+                    {item.value}
+                  </Text>
+                  <ToolTip content={item.label} placement="top" width="w-44 lg:w-52">
+                    <Icon name={IconNames.IC_INFO} className="h-3.5 w-3.5 text-darkCustom lg:h-5 lg:w-5" />
+                  </ToolTip>
+                </div>
+              </div>
+            ))}
+          </div>
+
+          <PageCard
+            className="border-0 px-4 py-6"
+            classNameSubTitle="mt-4"
+            gap={8}
+            subtitleMore={true}
+            title={t("TREES RESTORED")}
+            variantSubTitle="text-14-light"
+            subtitle={t(
+              `The numbers and reports below display data related to Indicator 1: Trees Restored described in <span class="underline">TerraFund’s  MRV framework </span>. Please refer to the linked MRV framework for details on how these numbers are sourced and verified.`
+            )}
+          >
+            <SecDashboard
+              title={t("Number of trees planted")}
+              type="legend"
+              secondOptionsData={LABEL_LEGEND}
+              data={NUMBER_OF_TREES_PLANTED}
+            />
+            <SecDashboard
+              title="Number of Trees Planted by Year"
```

Comment:
Since these views are not exclusive to admin, they need to be included in Transifex.

Reply:
done